src/library.cpp

```diff
@@ -135,7 +135,6 @@ QStringList Library::getLibraryZimsFromDir(QString dir) const
 
 void Library::loadMonitorDir(QString monitorDir)
```

Comment:
I wonder (not sure) if this should be renamed to `updateFromDir`

Reply:
Done